client/src/app/shared/shared-main/users/user-notifications.component.ts

```diff
@@ -19,6 +19,7 @@ export class UserNotificationsComponent implements OnInit {
   @Output() notificationsLoaded = new EventEmitter()
 
   notifications: UserNotification[] = []
+  sortField = 'createdAt'
```

Comment:
By default it should be `-createdAt` no? (newest first)

Reply:
Apparently no.